integrations/jira/jira/client.py

```diff
@@ -0,0 +1,116 @@
+import base64
+import typing
+
+import httpx
+from loguru import logger
+
+from jira.overrides import JiraResourceConfig
+from port_ocean.context.event import event
+
+PAGE_SIZE = 50
+WEBHOOK_NAME = "Port-Ocean-RT-Webhook"
```

Comment:
Rt?

Reply:
Supposed to be "real time" but I'll change to events